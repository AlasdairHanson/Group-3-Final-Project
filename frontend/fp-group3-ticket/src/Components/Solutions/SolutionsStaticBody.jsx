import React from "react";
import SolutionsBody from "./SolutionsBody";
import SolutionsHeader from "./SolutionsHeader";
import SolutionsSubHead from "./SolutionsSubHead";
const SolutionsStaticBody = () => {
  return (
    <>
      <div className="solutionsBody">
        <SolutionsHeader title="Ut posuere eu neque vel blandit. Nulla. " />
        <SolutionsSubHead title="Mauris pretium hendrerit lectus molestie aliquam. Sed. " />
        <SolutionsBody content="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod convallis metus ultricies tincidunt. Suspendisse potenti. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut maximus sem vel sapien aliquet, quis rhoncus arcu eleifend. Quisque in diam nunc. Phasellus et fringilla mi. Curabitur aliquet neque auctor accumsan sollicitudin. Vestibulum pellentesque ut dui sit amet mollis. Nulla at efficitur dolor, sit amet convallis nisl. Nulla auctor pulvinar nibh quis tempor. Donec auctor orci arcu, eget accumsan metus iaculis pulvinar. Duis porta sagittis dolor, nec vulputate urna. Nam venenatis consectetur lectus ut rhoncus. Phasellus ornare nunc eu luctus ultrices. Sed vitae dapibus neque. Maecenas dui tellus, cursus a dui ac, faucibus finibus dolor. Mauris eu turpis neque. Duis nulla nibh, egestas vel fringilla et, elementum non risus. " />
        <SolutionsHeader title="Ut posuere eu neque vel blandit. Nulla. " />
        <SolutionsSubHead title="Mauris pretium hendrerit lectus molestie aliquam. Sed. " />
        <SolutionsBody content="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod convallis metus ultricies tincidunt. Suspendisse potenti. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut maximus sem vel sapien aliquet, quis rhoncus arcu eleifend. Quisque in diam nunc. Phasellus et fringilla mi. Curabitur aliquet neque auctor accumsan sollicitudin. Vestibulum pellentesque ut dui sit amet mollis. Nulla at efficitur dolor, sit amet convallis nisl. Nulla auctor pulvinar nibh quis tempor. Donec auctor orci arcu, eget accumsan metus iaculis pulvinar. Duis porta sagittis dolor, nec vulputate urna. Nam venenatis consectetur lectus ut rhoncus. Phasellus ornare nunc eu luctus ultrices. Sed vitae dapibus neque. Maecenas dui tellus, cursus a dui ac, faucibus finibus dolor. Mauris eu turpis neque. Duis nulla nibh, egestas vel fringilla et, elementum non risus. " />
        <SolutionsHeader title="Ut posuere eu neque vel blandit. Nulla. " />
        <SolutionsSubHead title="Mauris pretium hendrerit lectus molestie aliquam. Sed. " />
        <SolutionsBody content="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod convallis metus ultricies tincidunt. Suspendisse potenti. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut maximus sem vel sapien aliquet, quis rhoncus arcu eleifend. Quisque in diam nunc. Phasellus et fringilla mi. Curabitur aliquet neque auctor accumsan sollicitudin. Vestibulum pellentesque ut dui sit amet mollis. Nulla at efficitur dolor, sit amet convallis nisl. Nulla auctor pulvinar nibh quis tempor. Donec auctor orci arcu, eget accumsan metus iaculis pulvinar. Duis porta sagittis dolor, nec vulputate urna. Nam venenatis consectetur lectus ut rhoncus. Phasellus ornare nunc eu luctus ultrices. Sed vitae dapibus neque. Maecenas dui tellus, cursus a dui ac, faucibus finibus dolor. Mauris eu turpis neque. Duis nulla nibh, egestas vel fringilla et, elementum non risus. " />
      </div>
    </>
  );
};
export default SolutionsStaticBody;
