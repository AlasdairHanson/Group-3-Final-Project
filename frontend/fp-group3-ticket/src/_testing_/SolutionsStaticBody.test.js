import { render, screen } from '@testing-library/react';
import SolutionsStaticBody from '../Components/Solutions/SolutionsStaticBody';
import '@testing-library/jest-dom';
import {create} from 'react-test-renderer';

describe(`Snapshot test of SolutionsStaticBody`, () => {

  test(` SolutionsStaticBody should match the snapshot every time it is rendered`, () => {
    const instance = create(<SolutionsStaticBody/>).toJSON(); 
    expect(instance).toMatchSnapshot();
  });
});