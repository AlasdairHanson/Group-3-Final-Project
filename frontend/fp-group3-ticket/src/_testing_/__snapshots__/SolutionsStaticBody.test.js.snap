// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test of SolutionsStaticBody  SolutionsStaticBody should match the snapshot every time it is rendered 1`] = `
<div
  className="solutionsBody"
>
  <h2
    className="solutionsHeader"
  >
    Ut posuere eu neque vel blandit. Nulla. 
  </h2>
  <h3>
    Mauris pretium hendrerit lectus molestie aliquam. Sed. 
  </h3>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod convallis metus ultricies tincidunt. Suspendisse potenti. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut maximus sem vel sapien aliquet, quis rhoncus arcu eleifend. Quisque in diam nunc. Phasellus et fringilla mi. Curabitur aliquet neque auctor accumsan sollicitudin. Vestibulum pellentesque ut dui sit amet mollis. Nulla at efficitur dolor, sit amet convallis nisl. Nulla auctor pulvinar nibh quis tempor. Donec auctor orci arcu, eget accumsan metus iaculis pulvinar. Duis porta sagittis dolor, nec vulputate urna. Nam venenatis consectetur lectus ut rhoncus. Phasellus ornare nunc eu luctus ultrices. Sed vitae dapibus neque. Maecenas dui tellus, cursus a dui ac, faucibus finibus dolor. Mauris eu turpis neque. Duis nulla nibh, egestas vel fringilla et, elementum non risus. 
  </p>
  <h2
    className="solutionsHeader"
  >
    Ut posuere eu neque vel blandit. Nulla. 
  </h2>
  <h3>
    Mauris pretium hendrerit lectus molestie aliquam. Sed. 
  </h3>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod convallis metus ultricies tincidunt. Suspendisse potenti. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut maximus sem vel sapien aliquet, quis rhoncus arcu eleifend. Quisque in diam nunc. Phasellus et fringilla mi. Curabitur aliquet neque auctor accumsan sollicitudin. Vestibulum pellentesque ut dui sit amet mollis. Nulla at efficitur dolor, sit amet convallis nisl. Nulla auctor pulvinar nibh quis tempor. Donec auctor orci arcu, eget accumsan metus iaculis pulvinar. Duis porta sagittis dolor, nec vulputate urna. Nam venenatis consectetur lectus ut rhoncus. Phasellus ornare nunc eu luctus ultrices. Sed vitae dapibus neque. Maecenas dui tellus, cursus a dui ac, faucibus finibus dolor. Mauris eu turpis neque. Duis nulla nibh, egestas vel fringilla et, elementum non risus. 
  </p>
  <h2
    className="solutionsHeader"
  >
    Ut posuere eu neque vel blandit. Nulla. 
  </h2>
  <h3>
    Mauris pretium hendrerit lectus molestie aliquam. Sed. 
  </h3>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod convallis metus ultricies tincidunt. Suspendisse potenti. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Ut maximus sem vel sapien aliquet, quis rhoncus arcu eleifend. Quisque in diam nunc. Phasellus et fringilla mi. Curabitur aliquet neque auctor accumsan sollicitudin. Vestibulum pellentesque ut dui sit amet mollis. Nulla at efficitur dolor, sit amet convallis nisl. Nulla auctor pulvinar nibh quis tempor. Donec auctor orci arcu, eget accumsan metus iaculis pulvinar. Duis porta sagittis dolor, nec vulputate urna. Nam venenatis consectetur lectus ut rhoncus. Phasellus ornare nunc eu luctus ultrices. Sed vitae dapibus neque. Maecenas dui tellus, cursus a dui ac, faucibus finibus dolor. Mauris eu turpis neque. Duis nulla nibh, egestas vel fringilla et, elementum non risus. 
  </p>
</div>
`;
